cmd/oras/internal/display/print.go

```diff
@@ -37,9 +37,10 @@ func Print(a ...any) error {
 // StatusPrinter returns a tracking function for transfer status.
 func StatusPrinter(status string, verbose bool) func(context.Context, ocispec.Descriptor) error {
 	return func(ctx context.Context, desc ocispec.Descriptor) error {
-		name, ok := desc.Annotations[ocispec.AnnotationTitle]
-		if !ok {
+		name := desc.Annotations[ocispec.AnnotationTitle]
+		if name == "" {
 			if !verbose {
+				// no status for unamed content
```

Comment:
Any reason to change the check?

Reply:
This is a bug that we fixed. For unnamed content, `ok` is always true but `name` is "". This causes verbose not to work.

Reply:
If that's the case, what caused `name` is set to empty string? We need to address that in the first place instead of having a workaround here.

Reply:
Name is empty is the definition of "unnamed content", which needs to be skipped when downloading. I think that's Billy's design.

Reply:
If the name is empty by default, then the name should never be set in the first place.